src/components/Compare/CompareTable.tsx

```diff
@@ -211,6 +212,55 @@ const CompareTable = (props: {
     trackCompareEvent(EventAction.OPEN_MODAL, 'Show All Locations');
   };
 
+  /**
+   * Show limited locations (100 max) in the following scenarios:
+   * - "Counties" tab selected on homepage.
+   * - "Metro areas" tab selected on homepage.
+   * - "USA" tab selected on location page.
+   */
+  const showLimitedLocations =
```

Comment:
i think this naming logic might be backwards? we show limited locations (100 max) when these conditions are false

Reply:
[as seen in the logic here](https://github.com/covid-projections/covid-projections/pull/3816/files#diff-d44f76b821573ad38ed21cd3b3888f013e0a64d6c1a813654fe2b5797a98c8feR229)

Reply:
True 😅